<!DOCTYPE html>
<html style="height: 100%">
<head>
  <meta content="text/html; charset=ISO-8859-1" http-equiv="content-type">
  <!---template modified from one provided by David Marcey and molvisions <http://www.molvisions.com/>	-helpful tips provided by David Marcey, Tim Driscoll, Angel Herraez, and Bob Hanson (on the Jmol Discussion list)-->
  <title>Shiga Toxin Type 2</title>
  <style>
	html {
	  margin: 0;
	  padding: 0;
    }
    body {
	  top: 0px;
	  left: 0px;
	  margin: 0px;
	  padding: 0px;
	  background-color: #ffffff;
	  color: #000000;
	  font-family: sans-serif;
	  font-size: 12px;
    }
    div.region {
	  position: absolute;
	  padding: 0px;
	  background-color: #ffffff;
	  color: #000000;
	  overflow: auto;
	  font-size: 14px;
    }
    div#navbar {
	  top: 0px;
	  left: 0px;
	  width: 100%;
	  height: 5%;
    }
    div#molecule  {
	  top: 5%;
	  left: 0px;
	  width: 60%;
	  height: 95%;
	  float: left
    }
    div#text {
	  top: 5%;
	  left: 61%;
	  width: 39%;
	  height: 95%;
    }
  </style>
  <script language="JavaScript" type="text/javascript" src="../jsmol/JSmol.min.js"></script>
  <script language="JavaScript">
    var loadScript = "zap; load 3gap.pdb; set specular on; moveTo 0 537 651 -537 113.9 120; set spin y 20; spin on; select all; labels off; restrict not water; spacefill on; color cpk; backbone off; ribbons off; cartoons off; wireframe off;";
    var Info = {
      width: 300, //"100%",
      height: 300, //"100%",
      script: loadScript,
      serverURL: "http://chemapps.stolaf.edu/jmol/jsmol/jsmol.php ",
      use: "HTML5 WEBGL",
      j2sPath: "../jsmol/j2s",
      isSigned: false,             // Java only
      jarFile: "JmolApplet.jar",  // Java only
      jarPath: "../jsmol/java",                // Java only
      memoryLimit: 512,            // Java only
      console: "jmolApplet_infodiv"
    }
    function ResetProteinButton() {Jmol.jmolButton(myJmol, loadScript, "Reset Protein");
    }
    function ResetComplexButton() {
      Jmol.jmolButton(myJmol, "zap; load 1cgp.pdb; moveTo 0 -205 -705 679 170.2 135; set spin y 20; spin on; select all; labels off; set specular on; wireframe off; color cpk; backbone off; ribbons off; cartoons off; spacefill on;","Reset Complex");
    }
  </script> 
  <link href="tutorials.css" rel="stylesheet" type="text/css">
</head>
<body link="#0000ff">
  <form name="baseForm" id="baseForm" action="" method="get">
  <div id="navbar" class="region">
    <font size="3">
      <strong><a href="http://biology.kenyon.edu/BMB/chime.htm">Biomolecules at Kenyon</a> &nbsp;
              <a href="http://biology.kenyon.edu/HHMI/index.htm">HHMI at Kenyon</a> &nbsp;
              <a href="http://jmol.sourceforge.net/">Jmol Home</a> &nbsp;
              <a href="http://biology.kenyon.edu/welcome.htm">Biology Dept</a> &nbsp;
              <a href="mailto:slonczewski@kenyon.edu">COMMENTS and CORRECTIONS</a>
      </strong>
    </font>
  </div>
  <div id="molecule" class="region">
    <script language="JavaScript" type="text/javascript">
      var frameWidth, frameHeight
        if (this.innerWidth) { frameWidth=this.innerWidth; frameHeight=this.innerHeight }
        else // IE
        {	
	      frameWidth = document.body.clientWidth
	      frameHeight = document.body.clientHeight
        }
      var appletSize = Math.min(frameWidth,frameHeight)
        Info.width = appletSize;
        Info.height = appletSize;
        myJmol = Jmol.getApplet("myJmol", Info);
    </script>
  </div>
  <div id="text" class="region">
    <center><h2><strong><a name="top" id="top"></a><i>E. coli</i> Shiga Toxin Type 2<br>
                 </strong><strong><font color="#660099" size="3"><br>
                 Rachel Martin '11 and Daniel Riggins '12</font></strong> 
            </h2>
    </center>
    </p>
    <hr>
      <h3> Contents:</h3>
      <ul>
        <a href="#INTRODUCTION">I. Introduction</a><br>
        <a href="#STRUCTURE">II. General Structure</a><br>
        <a href="#ADENINE">III. Adenine Binding</a><br>
        <a href="#ENTRY">IV. Mechanism of Entry into the Cell</a><br>
        <a href="#DIFFERENCES">V. Differences from Stx and Stx1</a><br>
        <a href="#REFERENCES">VI. References</a></strong>
      </ul>
      <hr>
      <p><a name="INTRODUCTION"></a> </p>
        <h3><strong>I. Introduction</strong></h3>
        Stx, or Shiga toxin, is a protein produced by the bacterium <i>Shigella dysenteriae</i>, and is named for Kiyoshi Shiga, who first discovered the microbes's role in dysentery<sup>[1]</sup>. A variation of this toxin, Stx2, is produced by <i>Escherichia coli</i> (most notably by strain O157:H7), and is one of the main sources of food poisoning<sup>[2]</sup>. Different strains of <i>E. coli</i> can also express Stx1, which causes milder illness than Stx2. Researchers are interested in how structural differences in these toxins affect virulence<sup>[3]</sup>. While Stx and Stx1 are nearly identical, the Stx2 protein structure contains some major alterations that seem to augment its pathological function.</p>
        Shiga toxins bind to receptors on the outside of the cell and are taken up into the endoplasmic reticulum (ER)<sup>[4].</sup> Because the receptors are not found in agricultural animals like cattle and swine, these animals can harbor toxic bacteria with little consequence. When toxin expressing <i>E. coli</i> are ingested by humans, they quickly enter host cells<sup>[2]</sup>. Once inside, Stx toxins act as <i>N</i>-glycosidases, depurinating specific adenines in the 60S subunit of ribosomes. Cleavage of adenine bases leads to incapacitation of ribosomes and termination of protein synthesis<sup>[4]</sup>.</p>
      <hr><a name="STRUCTURE"></a>
        <h3> <strong>II. General Structure</strong></h3>
        <script language="JavaScript">ResetProteinButton();</script>
        <p>Like other members of its toxin family, Stx2 is made up of one <span style="color: rgb(255, 0, 0);">alpha subunit</span> and five <span style="color: rgb(0, 0, 153);">beta subunits</span>. <script language="JavaScript">jmolButton("select all; labels off; spacefill off; color cpk; backbone off; ribbons off; cartoons on; wireframe off; select *a; color red; select atomno=190; label Alpha; font label 31; select *b, *c, *d, *e, *f; color blue; select atomno=2353; label Beta; font label 31; spin on;","");</script> The <span style="color: rgb(255, 0, 0);">alpha subunit</span> contains the <span style="color: rgb(56, 234, 138);">catalytic site</span> that cleaves <span style="color: rgb(204, 51, 204);">adenine</span> from the ribosome. <script language="JavaScript">jmolButton("select all; labels off; spacefill off; color cpk; backbone off; ribbons off; cartoons on; wireframe off; select *a; color red; select *b, *c, *d, *e, *f; color blue; spin off; moveto /* time, axisAngle */ 1.0 { -26 -914 -404 155.2} /* zoom, translation */ 182.5 -10.0 20.0 /* center, rotationRadius */ {8.417001 49.8325 5.140999} 49.747337 /* navigation center, translation, depth */ {0.0 0.0 0.0} -11.853653 92.764435 0.0; select ade; wireframe 30; spacefill 75; color magenta; spin off; delay 1; select arg170, val78, ser112, val162, ala166, tyr77, tyr 114; color mediumspringgreen; cartoon off; wireframe; delay .1; wireframe 30; spacefill 30; delay .1; spacefill 50; delay .1; spacefill 75; delay .1; spacefill 100; delay .1; spacefill 125; delay .1; spacefill 150; delay .1; spacefill 175; delay .1; spacefill 200; delay .1; spacefill 225; delay .1; spacefill 250; delay .1; spacefill 275; delay .1; spacefill 300; delay .1; spacefill 325; delay .1; spacefill 350; delay .1; spacefill 375; delay .1; spacefill 400; delay .1; spacefill; delay 1; moveto /* time, axisAngle */ 1.0 { -253 -923 -290 106.38} /* zoom, translation */ 277.57 -1.86 30.0 /* center, rotationRadius */ {8.417001 49.8325 5.140999} 49.747337 /* navigation center, translation, depth */ {0.0 0.0 0.0} -61.604683 140.31186 0.0; select *b, *c, *d, *e, *f; cartoon off;","");</script> The <span style="color: rgb(0, 0, 153);">pentameric ring</span> formed by the beta subunits binds to the membranes of target cells. All stx toxins preferentially bind to glycosphingolipids via fifteen active sites (three per <span style="color: rgb(2, 115, 207);">beta monomer</span>)<sup>[5]</sup>. <script language="JavaScript"> jmolButton("display protein, ade; select all; labels off; spacefill off; color cpk; backbone off; ribbons off; cartoons on; wireframe off; select *a; color red; select *b, *c, *d, *e, *f; color blue; select ade; wireframe 30; spacefill 75; color magenta; select arg170, val78, ser112, val162, ala166, tyr77, tyr 114; color mediumspringgreen; cartoon off; spacefill; spin off; moveto /* time, axisAngle */ 1.0 { -450 -891 -54 100.62} /* zoom, translation */ 138.0 -1.93 0.0 /* center, rotationRadius */ {8.417001 49.8325 5.140999} 49.747337 /* navigation center, translation, depth */ {0.0 0.0 0.0} -35.68257 63.773823 0.0; delay 1; select *b; color cyan; select atomno=2425; label One; font label 31; set labeloffset 5 -5; delay 0.5; select *b; color blue; select *c; color cyan; select atomno=2785; label Two; font label 31; delay 0.5; select *c; color blue; select *d; color cyan; select atomno=3793; label Three; font label 31; delay 0.5; select *d; color blue; select *e; color cyan; select atomno=3885; label Four; font label 31; set labeloffset 0 -5; delay 0.5; select *e; color blue; select *f; color cyan; select atomno=4878; label Five; font label 31; set labeloffset 0 -10; delay 0.5; select *f; color blue;","");</script></p>
        <p>The carboxy-terminal portion (<span style="color: rgb(255, 102, 102);">A2</span>)
of the <span style="color: rgb(255, 0, 0);">alpha
subunit</span> links it to the
pentameric
ring by extending through its center
pore. &nbsp;<span style="color: rgb(255, 102, 102);">A2</span>
traverses through it starting as an alpha helix,
then in an extended conformation. &nbsp;<span
 style="color: rgb(255, 102, 102);">A2</span>
ends as a terminal helix that projects
out of the <span style="color: rgb(0, 0, 153);">B-pentamer</span>
at 30 degrees {5}.&nbsp;
  <script language="JavaScript">
jmolButton("labels off; spacefill off; color cpk; backbone off; ribbons off; cartoons on; wireframe off; select *a; color red; select *b, *c, *d, *e, *f; cartoon on; color blue;select arg170, val78, ser112, val162, ala166, tyr77, tyr 114; color mediumspringgreen; spacefill; cartoon off; select ade; wireframe 30; spacefill 75; color magenta; spin off; moveto /* time, axisAngle */ 1.0 { 341 -874 -347 145.98} /* zoom, translation */ 138.0 -1.86 0.0 /* center, rotationRadius */ {8.417001 49.8325 5.140999} 49.747337 /* navigation center, translation, depth */ {0.0 0.0 0.0} 30.828058 28.93337 0.0; hide *b, *c, *d, *e, *f; select *a; select atomno>1905 and atomno<2225; color indianred; delay 0.25; color red; delay 0.25; color indianred; delay 0.25; color red; delay 0.25; color indianred; delay 0.5; select atomno=270; label One; font label 31; select atomno=2124; label Two; font label 31;","");
  </script>&nbsp;
  </strong><strong></strong></p>
  <p><strong><span style="color: rgb(255, 0, 0);">A1</span>
is
linked to <span style="color: rgb(255, 102, 102);">A2</span>
via the polypeptide chain and a <span style="color: rgb(102, 0, 204);">disulfide
bond</span> </strong><strong>between
the <span
 style="color: rgb(255, 255, 51); background-color: rgb(0, 0, 0);">sulphur
atoms</span> of
cysteine&nbsp;241 and cysteine&nbsp;260</strong><strong>.&nbsp;
  <script language="JavaScript">
jmolButton("select all; labels off; spacefill off; color cpk; backbone off; ribbons off; cartoons on; wireframe off; select *a; color red; select *b, *c, *d, *e, *f; color blue;select arg170, val78, ser112, val162, ala166, tyr77, tyr 114; color mediumspringgreen; spacefill; cartoon off; select ade; wireframe 30; spacefill 75; color magenta; spin off; select atomno>1905 and atomno<2225; color indianred; moveto /* time, axisAngle */ 1.0 { 566 -716 -408 131.43} /* zoom, translation */ 209.88 0.07 25.07 /* center, rotationRadius */ {8.417001 49.8325 5.140999} 49.747337 /* navigation center, translation, depth */ {0.0 0.0 0.0} 40.00118 27.544077 0.0; select cys 241, cys 260; color cpk; wireframe 40; color ssbonds purple;","");
  </script>&nbsp;
  </strong><strong>In
vivo, <span style="color: rgb(255, 0, 0);">A1</span>
separates from the rest of the toxin by nicking the
polypeptide chain and reducing a disulfide bond. &nbsp;This
structure shows the polypeptide
chain already broken.&nbsp;
Henceforth after breaking, <span style="color: rgb(255, 0, 0);">A1</span>
is active as an N-glycosidase. Unlike in the structure of Stx,
the
active site is accessible to small molecules before <span
 style="color: rgb(255, 0, 0);">A1</span>
parts from <span style="color: rgb(255, 102, 102);">A2</span>
{5}.<br>
  </strong></p>
        <hr><a name="ADENINE"></a>
        <h3> <strong>III. cAM</strong></h3>
        <script language="JavaScript">ResetProteinButton();</script>
        <p><strong>An important recognition site for cAMP within CAP is the
            ionic bond formed between the side chain of <font color="#0000FF">Arg-82</font>
            <script language="JavaScript">
  Jmol.jmolButton(myJmol, "select all; labels off; spacefill off; color cpk; ribbons off; cartoons off; wireframe off; backbone on; select cmp; backbone off; wireframe 0.6; select 82; color blue; backbone off; wireframe 0.6; move 0 0 0 60 0 0 0 0 2.5; move 0 0 0 0 0 0 0 0 1; move 0 0 0 -60 0 0 0 0 2.5;","")
</script> and the negatively charged <font color="#FFA500">phosphate </font>group
            of cAMP. In the crystal structure, the two cAMP molecules are buried
            deep within the <font color="#CC00FF">beta roll</font> and the <span

              style="color: #00FFFF; background-color:#000000;">C-helix</span>.
            <script language="JavaScript">
  Jmol.jmolButton(myJmol, "select all; labels off; spacefill off; color cpk; ribbons off; cartoons off; wireframe off; backbone on; select 108-131; backbone off; ribbons on; color cyan; select sheet and 19-100; backbone off; ribbons on; color magenta; select 70-100; color magenta; backbone off; select cmp; color lawngreen; backbone off; wireframe 0.6;","");
</script> It is unclear how cAMP enters or leaves the binding site, but this
            probably requires the separation of the two subunits of the dimer,
            or the movement of the beta roll and the C helix away from each
            other. Other side-chain interactions between the protein and cAMP
            are hydrogen bonds occuring at <span style="color: #FFFF00; background-color: #000000;">Thr-127,
              Ser-128, Ser-83</span>, and <span style="color: #FFFF00; background-color: #000000;">Glu-72</span>.
            <script language="JavaScript">
  Jmol.jmolButton(myJmol, "select all; labels off; spacefill off; color cpk; ribbons off; cartoons off; wireframe off; backbone on; select 82; color blue; backbone off; wireframe 1; select cmp; color lawngreen; backbone off; wireframe 0.6; select 127, 128, 83, 72; color yellow; backbone off; wireframe 0.6;","");
</script>Additional hydrogen bonding between is seen between cAMP and the
            polypeptide backbone at residues <font color="#FF0000">83</font>
            and <font color="#FF0000">71</font>
            <script language="JavaScript">
  Jmol.jmolButton(myJmol, "select all; labels off; spacefill off; color cpk; ribbons off; cartoons off; wireframe off; backbone on; select cmp; backbone off; wireframe 0.6; select 82; color blue; backbone off; wireframe 1; select 108-131; color cyan; backbone off; ribbons on; select 70-100; color magenta; backbone off; select cmp; color lawngreen; select protein; ribbons off; backbone on; color cpk; select 127, 128, 83, 72; color yellow; backbone off; wireframe 0.6; select 83, 71; color red; backbone off; wireframe 0.6;","");
</script></strong></p><strong>
          <hr><a name="ENTRY"></a>
          <h3> <strong>IV. DNA Binding</strong></h3>
          <script language="JavaScript">ResetComplexButton();</script>
          <p><strong>Once CAP has bound cAMP, it is ready to bind to the <a href="javascript:Jmol.script(myJmol, 'zap; load 1cgp.pdb; moveTo 0 -205 -705 679 170.2 135; set spin y 20; spin on; select all; labels off; set specular on; spacefill off; color cpk; backbone off; ribbons off; cartoons off; wireframe on;')">DNA</a>.
              Binding occurs at the conserved sequence of
              5'-AAATGTAGATCACATTT-3'
              <script language="JavaScript">
  Jmol.jmolButton(myJmol, "zoom 135; translate y 0; select all; labels off; spacefill off; color cpk; backbone off; ribbons off; cartoons off; wireframe on; select cmp; wireframe 0.6; color lawngreen; select a; wireframe 0.6; color lawngreen; select t; color orange; wireframe 0.6; select g; color yellow; wireframe 0.44; select c; color red; wireframe 0.6;","");
</script>Hydrogen bonds between the protein and the DNA phsophates occur at the
              backbone amide of <span style="color: #00FFFF; background-color:#000000;">residue
                139</span>, and the side chains of <span style="color: #00FFFF; background-color:#000000;">Thr-140,
                Ser-179,</span> and <span style="color: #00FFFF; background-color:#000000;">Thr-182</span>
              <script language="JavaScript">
  Jmol.jmolButton(myJmol, "zoom 135; translate y 0; select all; labels off; hbonds off; spacefill off; color cpk; ribbons off; cartoons off; wireframe off; backbone on; select cmp; backbone off; wireframe 0.6; color lawngreen; select dna; backbone off; wireframe on; select a; backbone off; wireframe 0.6; color lawngreen; select t; color orange; backbone off; wireframe 0.6; select g; color yellow; backbone off; wireframe 0.44; select c; color lawngreen; backbone off; wireframe 0.6; select 139, 140, 179, 182; color cyan; spacefill on; hbonds on; spin off; move 0 0 0 60 0 25 0 0 2.5; spin on;", "");
</script>In addition to these phosphate interactions, the side chains of <font

                color="#0000FF">Glu-181</font> and <font color="#0000FF">Arg-185</font>,
              both emanating from the <span style="color: #FFFF00; background-color: #000000;">recognition
                helix</span>
              <script language="JavaScript">
  Jmol.jmolButton(myJmol, "zoom 195; translate y 25; select all; labels off; hbonds off; spacefill off; color cpk; ribbons off; cartoons off; wireframe off; backbone on; select dna; backbone off; wireframe on; select 4, 5, 27, 28; backbone off; wireframe 0.6; hbonds on; select 177-193; backbone off; ribbons on; color yellow; select atomno=1644 or atomno=3587; label Recognition Helix; font label 31; select 181, 185; color blue; backbone off; wireframe 0.6;","");
</script>directly contact the bases within the major groove of the DNA. Because
              of the way that the protein binds to the DNA, a kink of ~40
              degrees occurs between nucleotide base pairs <span style="color: #FFFF00; background-color: #000000;">six</span>
              and <font color="#FF0000">seven</font> on each side of the dyad
              axis, 5'-<span style="color: #FFFF00; background-color: #000000;">T</span><font

                color="#FF0000">G</font>-3'
              <script language="JavaScript">
  Jmol.jmolButton(myJmol, "zoom 195; translate y 25; select all; labels off; hbonds off; spacefill off; color cpk; ribbons off; cartoons off; wireframe off; backbone on; select 177-193; color yellow; backbone off; ribbons on; select 181, 185; color blue; backbone off; wireframe 0.6; select dna; backbone off; wireframe on; select dna and (4, 5, 27, 28); backbone off; spacefill on; hbonds on;", "");
</script>This sequence has been shown to favor DNA flexibility and bending in
              other systems as well. Because of this kink, an additional five
              ionic interactions and four hydrogen bonds are able to occur
              between the protein and the DNA strand. Examples of these new
              interactions occur between <font color="#CC00FF">Lys-26,
                Lys-166, His-199</font> and the DNA sugar-phosphate backbone
              <script language="JavaScript">
  Jmol.jmolButton(myJmol, "zoom 195; translate y 25; select all; labels off; hbonds off; spacefill off; color cpk; backbone on; ribbons off; cartoons off; wireframe off; select dna; wireframe on; backbone off; select 26, 166, 199; color magenta; wireframe 0.8; backbone off; select t; wireframe on; backbone off;","");
</script>The DNA bend is integral to the mechanism of transcription activation.
              Not only does it place CAP in the proper orientation for
              interaction with RNA polymerase, but wrapping the DNA around the
              protein may result in direct contacts between upstream DNA and RNA
              polymerase.&nbsp;</strong></p>
          <hr><a name="DIFFERENCES"></a>
          <h3> <strong>V. Activating Regions</strong></h3>
          <script language="JavaScript">ResetComplexButton();</script>
          <p><strong>Transcription activation by CAP requires more than merely
              the binding of cAMP and binding and bending of DNA. CAP contains
              an "activating region" that has been proposed to participate in
              direct protein-protein interactions with RNA polymerase and/or
              other basal transcription factors. Specifically, amino acids <font

                color="#0000FF">156</font>, <font color="#FFA500">158</font>,<font

                color="#FF0000"> 159</font>, and <span style="color: #FFFF00; background-color: #000000;">162</span>
              <script language="JavaScript">
  Jmol.jmolButton(myJmol, "zoom 195; translate y 25; select all; labels off; hbonds off; spacefill off; color cpk; backbone on; ribbons off; cartoons off; wireframe off; select dna; wireframe 0.04; backbone off; select protein; backbone 100; select cmp; wireframe 0.6; backbone off; select 156; color blue; spacefill; backbone off; select 158; spacefill; backbone off; color orange; select 159; spacefill; backbone off; color red; select 162; spacefill; backbone off; color yellow;","");
</script>have been proposed to be critical for transcription activation by CAP.
              These amino acids are part of a surface loop composed of residues
              152-166
              <script language="JavaScript">
  Jmol.jmolButton(myJmol, "zoom 195; translate y 25; select all; labels off; hbonds off; spacefill off; color cpk; backbone on; ribbons off; cartoons off; wireframe off; select dna; wireframe 0.04; backbone off; select 156, 158, 159, 162; wireframe 0.6; backbone off;","");
</script>Researchers have concluded that the third and final step in
              transcription activation is this direct protein-protein contact
              between amino acids 156-162 of CAP, and RNA polymerase.</strong></p>
          <br>
          <hr><a name="REFERENCES"></a>
          <h3> <strong>VI. References</strong></h3>
          <strong><font size="-1">Gunasekera, Angelo, Yon W. Ebright, and
              Richard H. Ebright. 1992. DNA Sequence Determinants for Binding of
              the <i>Escherichia coli</i> Catabolite Gene Activator Protein. <i>The
                Journal of Biological Chemistry</i> 267:14713-14720.</font></strong>
          <p><strong><font size="-1">Schultz, Steve C., George C. Shields, and
                Thomas A. Steitz. 1991. Crystal Structure of a CAP-DNA complex:
                The DNA Is Bent by 90 degrees <i>Science</i> 253: 1001-1007.</font></strong>
          </p>
          <p><strong><font size="-1">Vaney, Marie Christine, Gary L. Gilliland,
                James G. Harman, Alan Peterkofsky, and Irene T. Weber. 1989.
                Crystal Structure of a cAMP-Independent Form of Catabolite Gene
                Activator Protein with Adenosine Substituted in One of Two
                cAMP-Binding Sites. <i>Biochemistry</i> 28:4568-4574.</font></strong>
          </p>
          <p><strong><font size="-1">Weber, Irene T., Gary L. Gilliland, James
                G. Harman, and Alan Peterkofsky. 1987. Crystal Structure of a
                Cyclic AMP-independent Mutant of Catabolite Activator Protein. <i>The
                  Journal of Biological Chemistry</i> 262:5630-5636.</font></strong>
          </p>
          <p><strong><font size="-1">Zhou, Yuhong, Ziaoping Zhang, and Richard
                H. Ebright. 1993. Identification of the activating region of
                catabolite gene activator protein (CAP): Isolation and
                characterization of mutants of CAP specifically defective in
                transcription activation. <i>Proceedings of the National
                  Academy of Sciences of the United States of America</i>
                90:6081-6085.</font></strong> </p>
          <p><strong><a href="#top">Back to Top</a></strong></p>
        </strong></div>
      <strong> </strong></form>
  </body>
</html>
